src/sentry/conf/server.py

```diff
@@ -1138,6 +1138,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:issue-alert-fallback-targeting": False,
     # Enable removing issue from issue list if action taken.
     "organizations:issue-list-removal-action": False,
+    "organizations:issue-list-saved-searches-button-location": False,
```

Comment:
maybe add a comment explaining what the feature flag does?

Reply:
Ah yes good catch!